README.md

```diff
@@ -11,27 +11,54 @@ Here is some material to read up on DLCs and ItchySats' protocol implementation:
 - [Blogpost](https://comit.network/blog/2022/01/11/cfd-protocol-explained) by @luckysori of the COMIT team, describing the protocol in detail
 - [DLC overview](https://bitcoinops.org/en/topics/discreet-log-contracts/) on Bitcoin Optech for information beyond ItchySats
 
+## Installation
+
+### Bitcoin/Lightning nodes
+
+ItchySats is available on many DIY Bitcoin/Lightning node distributions:
+
+- [RaspiBlitz](https://github.com/rootzoll/raspiblitz#feature-overview)
+- [Umbrel](https://getumbrel.com/)
+- [Citadel](https://runcitadel.space/)
+
+### Desktop App
+
+We package ItchySats as an Electron app for Windows and MacOS.
+Latest release can be found [here](https://github.com/itchysats/itchysats/releases).
```

Comment:
You could link to latest: `https://github.com/itchysats/itchysats/releases/latest`